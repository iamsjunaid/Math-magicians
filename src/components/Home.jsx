const Home = () => (
  <div className="home">
    <h2>Homepage</h2>
  </div>
);
export default Home;
