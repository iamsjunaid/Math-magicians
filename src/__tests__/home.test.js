import { render } from '@testing-library/react';
import Home from '../components/Home';

test('renders Home correctly', () => {
  const { container } = render(
    <Home />,
  );
  expect(container).toMatchSnapshot();
});
