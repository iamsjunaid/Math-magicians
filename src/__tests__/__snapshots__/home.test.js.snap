// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FetchQuote correctly 1`] = `
<div>
  <div
    class="home"
  >
    <section
      class="content"
    >
      <h2>
        Welcome to our page!
      </h2>
      <p>
        Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
      </p>
      <ul>
        <li>
          ☑️Read a random math-related quote.
        </li>
        <li>
          ☑️Make simple calculations.
        </li>
      </ul>
    </section>
    <section
      class="img-container"
    >
      <img
        alt="math"
        src="https://i.gifer.com/G0yR.gif"
      />
    </section>
  </div>
</div>
`;
